destination.go

```diff
@@ -36,60 +36,59 @@ func (d *Destination) Parameters() map[string]sdk.Parameter {
 	return map[string]sdk.Parameter{
 		Servers: {
 			Default:     "",
-			Required:    true,
+			Validations: []sdk.Validation{sdk.ValidationRequired{}},
 			Description: "A list of bootstrap servers to which the plugin will connect.",
 		},
 		Topic: {
 			Default:     "",
-			Required:    true,
+			Validations: []sdk.Validation{sdk.ValidationRequired{}},
 			Description: "The topic to which records will be written to.",
 		},
 		Acks: {
 			Default:     "all",
-			Required:    false,
 			Description: "The number of acknowledgments required before considering a record written to Kafka. Valid values: none, one, all.",
 		},
 		DeliveryTimeout: {
 			Default:     "10s",
-			Required:    false,
 			Description: "Message delivery timeout.",
 		},
 		ClientCert: {
 			Default:     "",
-			Required:    false,
 			Description: "A certificate for the Kafka client, in PEM format. If provided, the private key needs to be provided too.",
 		},
 		ClientKey: {
 			Default:     "",
-			Required:    false,
 			Description: "A private key for the Kafka client, in PEM format. If provided, the certificate needs to be provided too.",
 		},
 		CACert: {
 			Default:     "",
-			Required:    false,
 			Description: "The Kafka broker's certificate, in PEM format.",
 		},
 		InsecureSkipVerify: {
-			Default:  "false",
-			Required: false,
+			Default: "false",
 			Description: "Controls whether a client verifies the server's certificate chain and host name. " +
 				"If `true`, accepts any certificate presented by the server and any host name in that certificate.",
 		},
 		SASLMechanism: {
 			Default:     "",
-			Required:    false,
 			Description: "SASL mechanism to be used. Possible values: PLAIN, SCRAM-SHA-256, SCRAM-SHA-512.",
 		},
 		SASLUsername: {
 			Default:     "",
-			Required:    false,
 			Description: "SASL username. required if saslMechanism is provided.",
 		},
 		SASLPassword: {
 			Default:     "",
-			Required:    false,
 			Description: "SASL password. required if saslMechanism is provided.",
 		},
+		Compression: {
+			Default:     "",
+			Description: "Compression applied to messages. Possible values: gzip, snappy, lz4, zstd.",
```

Comment:
Would be nice if we attach a [`ValidationInclusion`](https://github.com/ConduitIO/conduit-connector-sdk/blob/edae165f49975c04fb5d9e549ea1987060068fc2/validation.go#L131) on this parameter, then the UI displays a dropdown. We should also specify the parameter [type](https://github.com/ConduitIO/conduit-connector-sdk/blob/edae165f49975c04fb5d9e549ea1987060068fc2/specifier.go#L51).

Reply:
Yeah, I was thinking about that too, but I'd like to go over all of the configs and update them all in one go.

Reply:
Fine with me 👍

Reply:
At that point we could also move to `paramgen` and simplify the parsing of parameters using `sdk.Util.ParseConfig`.